doc/logo.md

```diff
@@ -0,0 +1,177 @@
+
+Our goal is to make that early Logo systems available to wider auditory of
+enthusiasts for exploration, experimenting and, of course, hacking.
+
+### BBN Logo
+
+BBN PDP-10 LOGO system was implemented originally by Walter B. Weiner with
+the assistance of Paul Wexelblat and Charles R. Morgan at BBN. 
+We have source file with a copyright from 1970. It is version modified at National
+Research Council, Ottawa, Canada by A.G. Smith and R.A.Orchard.
+That version of BBN Logo for PDP-10 TOPS-10 and TENEX was assembled with MACRO-10
+in ITS, and dumped with the DECUUO bootstrap to get ITS executable binary by Lars
+Brinkhoff. 
+
+to run, type:
+`:bbn;logo`
+
+Here is some BBN Logo related documentation:
+
+ Bolt Beranek and Newman, Inc., Report no. 2165 vol. 4 "The Logo Processor: A Gide 
+ for System Programmers", pub date 30 June 1971.
+
+ Bolt Beranek and Newman, Inc., Report no.1889 "Programming-Languages as a Conceptual 
+ Framework for Teaching Mathematics" (Feurzeig, Papert at al., 1969)
+ Appendix "A Description of the Logo Language and System".
+
+ Both documents are available at:
+
+ https://github.com/PDP-10/bbn-logo
+ 
+### MIT CLOGO
+
+MIT CLOGO is a direct BBN LOGO descendant. It is, in essence, BBN Logo ported to ITS
+and MIDAS and enhanced with turtle graphics and other capabilities at MIT.
+
+We have pretty stable CLOGO version 49 binary as well as good-commented source code for
+the version.
+
+Unfinished documentation for the CLOGO may be reached at:
+https://github.com/PDP-10/its/blob/alexey/CLOGO/doc/_info_/clogo.manual
+
+### MIT LISP LOGO
+
+LISP LOGO is an implementation of the LOGO language written in MACLISP for the ITS, TEN50
+and TENEX PDP-10 systems, and MULTICS. The system were implemented by Ira Goldstein,
+Henry Lieberman in the early 70's at MIT.
+One of the reason for this implementation was to provide a natural transition to the more
+powerful computational world of LISP as the Logo programmer grows more sophisticated.
+When desired, the one has access to all of the capabilities of MACLISP.
+
+As far as we know there were several LISP LOGO versions:
+- LLOGO - Standard version which uses a vocabulary which is compatible with 11LOGO.
+- CLLOGO - A version which uses a vocabulary which is compatible with CLOGO.
+- NLLOGO - An experimental version of LISP LOGO.
+- ELLOGO - Another experimental version of LISP LOGO.
+- BWLOGO - LISP LOGO with black and white graphics.
+
+LISP LOGO had available several packages of special functions:
+- TV TURTLE - LISP LOGO package provides Knight TV system Logo Graphics.
+- DISPLAY TURTLE - LISP LOGO package provides 340 / GT40 display Logo Graphics.
+- GERMLAND - some kind of display turtle for character displays without grahics capabilities.
+- MUSIC BOX - the package provides Logo music capabilities.
+
+The LLOGO variant of LISP LOGO was recently brought back to life during several amazing
+hacking session performed by Eric Swenson and Lars Brinkhoff. As a result, we
```

Comment:
No need to praise Swenson and Brinkhoff here.

Reply:
I do not quite agree, but I obey.